app/assets/stylesheets/shared/comments.scss

```diff
@@ -35,7 +35,6 @@
         font-size: 16px;
         line-height: 30px;
         padding-left: 5px;
-        width: 50%;
```

Comment:
This change is breaking the default view. If your name is too long, the date will be in the next row. 
To change only shared view, add `width: auto;` inside of `&.simple` and `.comment-name {`, line `289`